.gitlab-ci.yml

```diff
@@ -9,6 +9,7 @@ variables:
   GIT_SUBMODULE_STRATEGY: recursive
   PROJECT_ALLOC_NAME: ${CI_PROJECT_NAME}_ci_${CI_PIPELINE_ID}
   BUILD_ROOT: ${CI_PROJECT_DIR}
+  ALLOC_BANK: eng
```

Comment:
Have you looked into whether "eng" is appropriate for our use. Maybe it doesn't matter.